supervision/utils/video.py

```diff
@@ -141,6 +673,7 @@ def _validate_and_setup_video(
     return video, start, end
 
 
+@DeprecationWarning
 def get_video_frames_generator(
```

Comment:
Internals of this deprecated function should be now reimplemented using new `Video` API.